packages/jobs/lib/processor/processor.worker.ts

```diff
@@ -84,7 +85,7 @@ export class ProcessorChild {
 
     async start(): Promise<void> {
         logger.info(`Starting Processor: ${JSON.stringify(this.opts)}`);
-        this.processor.start();
+        this.processor.start({ tracer });
```

Comment:
you sure you need to pass the tracer? I don't think it's necessary

Reply:
It is not entirely necessary but since it is used in the orchestrator library I wanted to make it clear where the Tracer is coming from